content/scout/release-notes.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Docker Scout release notes
+description: Learn about the latest features of Docker Scout
+keywords: docker scout, release notes, changelog, features, changes, delta, new, releases
+---
+
+This page contains information about the new features, improvements, known
+issues, and bug fixes in Docker Scout releases. These release notes cover the
+Docker Scout platform, including the Dashboard. For CLI release notes, refer to
+the `docker/scout-cli` [GitHub repository](https://github.com/docker/scout-cli/releases).
+
+Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1)
+for what's coming next.
+
+## 20nn-nn-nn
+
+This marks the General Availability (GA) release of Docker Scout.
+
+### New
+
+The following new features are included in this release:
+
+- [Policy Evaluation](#policy-evaluation) (Early Access)
+- [Amazon ECR integration](#amazon-ecr-integration)
+- [Sysdig integration](#sysdig-integration)
+- [JFrog Artifactory integration](#jfrog-artifactory-integration)
+
+#### Policy evaluation
+
+Policy Evaluation is an early access feature that helps you ensure software
+integrity and track how your artifacts are doing over time. This release ships
+with four out-of-the-box policies, enabled by default for all organizations.
+
+![Policy overview in Dashboard](./images/release-notes/policy-ea.webp)
+
+- **Base images not up-to-date** evaluates whether the base images are out of
+  date, and require updating. Up-to-date base images help you ensure that your
+  environments are reliable and secure.
+- **Critical and high vulnerabilities with fixes** reports if there are
+  vulnerabilities with critical or high severity in your images, and where's
+  there a fix version available that you can upgrade to.
```

Comment:
```suggestion
  there is a fix version available that you can upgrade to.
```

Reply:
oops, thanks for catching, this should read `and where there's a fix version...` not `and where's there` 😭 🤣